daemon/logger/awslogs/cloudwatchlogs.go

```diff
@@ -88,7 +89,11 @@ func init() {
 // the EC2 Instance Metadata Service.
 func New(ctx logger.Context) (logger.Logger, error) {
 	logGroupName := ctx.Config[logGroupKey]
-	logStreamName := ctx.ContainerID
+	logStreamName, err := loggerutils.ParseLogTag(ctx, loggerutils.DefaultTemplate)
```

Comment:
Is the use-case for setting `--log-opt tag="{{.Name}}/{{.ID}}"` (or similar) on the daemon so that it provides the template by default?  If so, can we get the docs updated as well?